src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/RefreshOrchestrator.java

```diff
@@ -78,14 +81,20 @@ public void validate(Long jobId, UserRequest userRequest) {
             throw new BadRequestException("Invalid JobId request");
         }
 
-
         if (userRequest != null && nullCheckPredicate.test(userRequest.getUserIds())) {
-            //Extract and Validate received users List
+            // Extract and Validate received users List
             parseRequestService.validateUserRequest(userRequest);
             log.info("Validated userIds {}", userRequest.getUserIds());
         }
+
+        // Ensure only permitted service is invoking caseworker refresh API
```

Comment:
Can we move this block to the start of the validate function: as we should fail with this first rather than other messages..